src/loaders/express.js

```diff
@@ -109,28 +109,22 @@ module.exports = async (app) => {
 
   app.use(checkAuthExpiredMiddleware);
 
-  ['v1', 'v2'].forEach((version) => {
-    // api and api.v2 could be renamed to api.v1 api.v2 once we focus on data model,
-    // for now leaving like this so we don't get merge conflicts
-    // with other changes that may come in on "api/"
-    const versionName = version === 'v1' ? 'api' : `api.${version}`;
-
-    app.use(
-      OpenApiValidator.middleware({
-        apiSpec: path.join(__dirname, '..', 'specs', `${versionName}.yaml`),
-        validateRequests: true,
-        validateResponses: {
-          onError: (error) => {
-            console.log('Response body fails validation: ', error);
-
-            AWSXRay.getSegment().addMetadata('openApiValidationError', JSON.stringify(error));
-            AWSXRay.getSegment().addAnnotation('openApiValidationFailed', true);
-          },
+  const versionName = 'api.v2';
```

Comment:
is this still needed? or i'ts supposed to be cleaned up in the worker/pipeline cleanup?

Reply:
this is where the api.yaml spec is defined. We use only one now so I guess i can remove this variable